files/config/default/etc/email/screencast_publication

```diff
@@ -0,0 +1,281 @@
+        <!-- pre-header -->
+        <table style="display:none!important;">
+            <tr>
+                <td>
+                    <div style="overflow:hidden;display:none;font-size:1px;color:#ffffff;line-height:1px;font-family:Arial;maxheight:0px;max-width:0px;opacity:0;">
+                        Pre-header
+                    </div>
+                </td>
+            </tr>
+        </table>
+        <!-- pre-header end -->
+        <!-- header -->
+        <!-- end header -->
+
+        <!-- content section -->
+        <table border="0" width="100%" cellpadding="0" cellspacing="0" bgcolor="ffffff" class="bg_color">
+
+            <tr>
+                <td align="center">
+                    <table border="0" align="center" width="640" cellpadding="0" cellspacing="0" class="container590">
+                        <tr>
+                            <td align="center">
+                                <table width="640"  border="0" align="center" cellpadding="0" cellspacing="0">
+                                    <tr>
+                                        <td align="left" style="color: #333; font-size: 14px; font-family: 'Work Sans', Calibri, sans-serif; line-height: 24px;">
+                                            <div style="line-height: 24px">
+                                                <p>Dear ${catalogs['series']['creator']},</p>
+                                                <br/>
+                                                <p>
+                                                    <span>The following recording was uploaded your </span>
+                                                    <#if workflowConfig['site_id']?has_content>
+                                                        <a href="https://vula.uct.ac.za/portal/site/${workflowConfig['site_id']}">Vula site.</a>
+                                                    <#else>
+                                                        <a href="https://vula.uct.ac.za/">Vula site.</a>
+                                                    </#if>
+                                                </p>                                                
+                                                <p>Event: <b>${catalogs['episode']['title']}</b></p> 
+                                                <br/>
+                                                <#if catalogs['episode']['created']?has_content>
+                                                    <#setting time_zone="Africa/Johannesburg">
+                                                    <#assign eventDate = catalogs['episode']['created']?substring(0,9)>
+                                                    <#assign eventTime = catalogs['episode']['created']?substring(11,19)>
+                                                    <p>Date & time: <b>${(eventDate + ' ' + eventTime + ' UTC')?datetime('yyyy-MM-dd HH:mm:ss z')}</b></p>
+                                                <#else>
```

Comment:
It's never going to be a scheduled event, so why have the else statement?

Reply:
That's true, we won't need any exception handling here as the create date will always be available. Removing this.